### PR TITLE
TST: move more series tests to test_arithmetic

### PR DESCRIPTION
**Pull request by jbrockmendel**
Doesn't overlap with other outstanding PRs.

Per usual, there is more that _could_ be done, but we're doing this in pieces.

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/series/test_arithmetic.py [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-from pandas import Series, Timestamp, Period
+from pandas import Series, Timestamp, Timedelta, Period, NaT
 from pandas._libs.tslibs.period import IncompatibleFrequency
 
 import pandas as pd
 import pandas.util.testing as tm
 
 
+@pytest.fixture
+def tdser():
```
Comment (jreback) [inline pandas/tests/series/test_arithmetic.py]: add a doc-string

File: pandas/tests/series/test_arithmetic.py [hunk 2]
```diff
@@ -377,3 +488,361 @@ def test_dt64ser_sub_datetime_dtype(self):
...
+        with pytest.raises(TypeError):
+            tdser + vector
+        with pytest.raises(TypeError):
+            vector + tdser
+        with pytest.raises(TypeError):
+            tdser - vector
+        with pytest.raises(TypeError):
+            vector - tdser
+
+    # ------------------------------------------------------------------
```
Comment (jreback) [inline pandas/tests/series/test_arithmetic.py]: these are nice comments / breaks, use liberally (as you are doing here, but elsewhere as well)

**Review by jreback — CHANGES_REQUESTED** [review]
small comment. we have *so* many tests that unfortunately some of these test files are still big, but not sure can do much about this.

**Comment by jreback** [comment]
ping when pushed on comment.

**Review by jreback — APPROVED** [review]

**Comment by jbrockmendel** [comment]
Ping

**Comment by jreback** [comment]
thanks!

**Pull request merged**